docs/content/Caching/Overview.md

```diff
@@ -28,112 +28,36 @@ unless it is necessary. To speed up query performance, consider using
 
 ## Pre-Aggregations
 
-The **pre-aggregation** engine builds a layer of aggregated data in your
-database during runtime and maintains it to be up-to-date.
+Pre-aggregations is the layer of the aggregated data built and refreshed by Cube.js. It can dramatically improve the queries performance and provide a higher concurrency.
```

Comment:
```suggestion
Pre-aggregations is a layer of the aggregated data built and refreshed by Cube.js. It can dramatically improve the query performance and provide a higher concurrency.
```